bazelisk.go

```diff
@@ -634,6 +638,12 @@ func main() {
 		}
 	}
 
+	// print bazelisk version information if "version" is the first argument
+	// bazel version is executed after this command
+	if len(args) > 0 && args[0] == "version" {
+		fmt.Printf("Bazelisk version: %s\n", BazeliskVersion)
```

Comment:
We should handle the case where `--gnu_format` is passed to `bazel version`. In that case, we have to print a slightly different version syntax: https://www.gnu.org/prep/standards/html_node/_002d_002dversion.html

Maybe like this:
```
$ bazel version --gnu_format
Bazelisk 0.0.5
Bazel 0.26.0
```

Reply:
Thanks, I didn't know about the `--gnu_format` argument, I'll implement it. 👍